projects/elife.yaml

```diff
@@ -2047,12 +2050,40 @@ fastly-logs:
 
 data-pipeline:
     description: Data consolidation project
-    domain: false
-    intdomain: false
+    domain: pipeline
+    intdomain: pipeline
+    formula-repo: https://github.com/elifesciences/data-pipeline-formula
     aws:
-        ec2: false
+        ec2:
+            ami: ami-1d4e7a66 # 16.04
+        ports:
+            - 22
+            - 443
         s3:
```

Comment:
this environment exists:
https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stack/detail?stackId=arn:aws:cloudformation:us-east-1:512686554592:stack%2Fdata-pipeline--demo%2F238635c0-970c-11e8-9a6a-50d5cd26c2d2
You can either remove it and create new ones, or run `update_infrastructure` to add the new resources.

Reply:
or we could leave it be or destroy it. @de-code , what is using the `demo` instance?

Reply:
It was only there for the initial prototype. One was the _inbox_, from which it was moved to the _archive_. It kind of makes the backfill straightforward (just copy it back into the _inbox_). Personally I'd prefer read-only access to the input.

Reply:
:+1: for destroying and recreating? The buckets will be left around by the current deletion strategy I think, need to be manually cleaned up.